Dockerfile

```
@@ -0,0 +1,4 @@
+FROM eclipse-temurin:25 as runner
+WORKDIR runner
```

Hadolint (2.14.0)

[error] Use absolute WORKDIR

(DL3000)